src/core/Instances.tsx

```diff
@@ -132,10 +119,11 @@ const Instances = React.forwardRef(
 
     return (
       <instancedMesh
+        userData={{ instances }}
         matrixAutoUpdate={false}
-        ref={mergeRefs([ref, parentRef])}
+        ref={mergeRefs([ref as any, parentRef])}
```

Comment:
you shouldn't need to do this. What happens if you type `forwardRef` in a different way:

```ts
const Instances = React.forwardRef<THREE.InstancedMesh, InstancesProps>(
  (
    { children, range, limit = 1000, frames = Infinity, ...props },
    ref
  )
```

`any` is bad practice imo

Reply:
i don't know why, it works locally, it crashes on the vercel ci. tsc ran without problems here.

Reply:
btw here is the fallout https://vercel.com/pmndrs/drei/HYTZjPhdCzNsbgsZpon9RvSUJQUZ

Reply:
@joshuaellis tried your suggestion but nothing: https://vercel.com/pmndrs/drei/P7fEUjFYSSRgLUgkps5K8dMakAcS

locally it works.

Reply:
removing the "THREE." did the trick